client/src/components/BallotVerifierContest.vue

```diff
@@ -204,10 +204,10 @@ export default defineComponent({
 }
 .BallotVerifierContest__Option_Crosses {
   grid-area: crosses;
-  padding: 0.5rem 1rem;
+  padding: 1rem;
   background-color: #f8f9fa;
   border-top: #dddddd 1px solid;
-  justify-content: end;
+  direction: rtl;
```

Comment:
How will this work with the upcoming feature of rlt designs in general? Most people in the world are still right-handed regardless of their reading direction.

Reply:
This is just to align the crosses to the left and should work for a CSS grid as a justify-content: flex-end on flexbox. It shouldn't interfere anywhere else.

Reply:
Apparently, this is the only solution for this specific case. But I checked and it does not have any effects on a future `rtl` global configuration (f.ex. to support arabic).